<ui version="4.0" >
 <class>GetMargin</class>
 <widget class="QDialog" name="GetMargin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>207</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>310</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>GetMargin Padding</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Margin/Padding</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Left</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3" >
              <property name="suffix" >
               <string> pt.</string>
              </property>
              <property name="decimals" >
               <number>4</number>
              </property>
              <property name="minimum" >
               <double>-50.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>100000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Top</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_4" >
              <property name="suffix" >
               <string> pt.</string>
              </property>
              <property name="decimals" >
               <number>4</number>
              </property>
              <property name="minimum" >
               <double>-50.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>100000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Bottom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox" >
              <property name="suffix" >
               <string> pt.</string>
              </property>
              <property name="decimals" >
               <number>4</number>
              </property>
              <property name="minimum" >
               <double>-50.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>100000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Right</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2" >
              <property name="suffix" >
               <string> pt.</string>
              </property>
              <property name="decimals" >
               <number>4</number>
              </property>
              <property name="minimum" >
               <double>-50.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>100000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Unit:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox" >
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <item>
          <property name="text" >
           <string>mm</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>cm</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>inch</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>pt</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>em</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>dm</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>pi</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>dd</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>cc</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>in</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2" >
         <property name="text" >
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../osApplication.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>GetMargin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>165</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel" >
     <x>72</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>GetMargin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>260</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel" >
     <x>304</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
